hw/top_earlgrey/data/ip/chip_uart_testplan.hjson

```diff
@@ -0,0 +1,252 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_uart
+  testpoints: [
+    // Basic smoke tests:
+    {
+      name: chip_sw_uart_tx_rx_smoke
+      desc: '''Verify the transfer of data from UART
+
+               - Powerup OT and enable UART IO.
```

Comment:
I'd probably drop the "powerup OT" prefix from these descriptions. I think we can probably assume that a test will require the chip to turn on :-)

Reply:
Removed `powerup OT`